### PR TITLE
Fix several issues

### PR DESCRIPTION
**Pull request by carter-thaxton**
Hello.  I tried to use your `golegal` program, and discovered that I could not run it out of the box.  Here are some fixes I made in order to let `golegal` be used more generally by others.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 legal
 start
+3.0/
```
Comment (carter-thaxton) [inline .gitignore:3]: Ideally we'd put all results into a directory like `results`, and then ignore that.  For now, just ignore the most common target generated by `make`.

**Comment by tromp** [comment]
Thanks for fixes!

File: .gitignore (hunk 1 above)
Comment (tromp) [inline .gitignore:3]: Thanks for the fixes, Carter.
Were you just curious to see the software run,
or did you actually want to verify some of the results?

regards,
-John

**Comment by carter-thaxton** [comment]
Well, I came across your site at https://tromp.github.io/go/legal.html, and was just curious about the implementation.  Great work!

**Comment by tromp** [comment]
dear Carter,

> Well, I came across your site at https://tromp.github.io/go/legal.html, and was just curious about the implementation. Great work!

Thanks! I'm busy making slides for a presentation about it.
Will you by any chance be at the US Go Congress in Boston?

regards,
-John

**Comment by carter-thaxton** [comment]
>  Will you by any chance be at the US Go Congress in Boston?

Alas no.  I'm on the west coast.  Best of luck on the presentation!

**Pull request merged**